lib/utils/computed-property.js

```diff
@@ -51,18 +51,20 @@ function isCPAccessor(node) {
   }
 
   let callee = parent.callee;
-  if (callee.type === 'Identifier' && callee.name === 'computed') {
+  if (callee.type === 'Identifier' && (callee.name === 'computed' || callee.name === importedCPName)) {
     return true;
   }
 
   if (callee.type === 'MemberExpression') {
     let caller = getCaller(callee);
-    return caller === 'Ember.computed' || caller === 'Em.computed';
+    return caller === 'Ember.computed' || caller === 'Em.computed' || caller === importedEmberName + '.computed';
   }
   return false;
 }
 
-module.exports = function isCPGetter(node) {
-  return isCPDesc(node, 'get') || isCPAccessor(node) || isPrototypeExtCP(node);
+module.exports = function isCPGetter(node, importedCPName, importedEmberName) {
```

Comment:
For some reason it feels more natural to have `importedEmberName` come  before `importedCPName`, like in hierarchical order.